internal/common/baseclient/iifo.go

```diff
@@ -0,0 +1,157 @@
+package baseclient
+
+import (
+	"math"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+// Disaster defines the disaster location and magnitude.
+// These disasters will be stored in PastDisastersDict (maps round number to disaster that occurred)
+// TODO: Agree with environment team on disaster struct representation
+type DisasterInfo struct {
+	CoordinateX float64
+	CoordinateY float64
+	Magnitude   float64
+	TurnNumber  int
+}
+
+// PastDisastersDict is a helpful construct for
```

Comment:
? what is int?

Reply:
int is a type? that is used for the storing the turnNumber

Reply:
```
// DisasterReport encapsulates a disaster location and magnitude. Note: magnitude of 0 => no disaster
type DisasterReport struct {
	magnitude, x, y float64
}
```
This was our initial definition so your `DisasterInfo` looks good